src/ImageSharp/Colors/PackedPixel/IPixel.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace ImageSharp
+{
+    using System;
+    using System.Numerics;
+
+    public interface IPixel
```

Comment:
The only direct reference to the property is here

https://github.com/MonoGame/MonoGame/blob/7c3d6870a38f8a5e479e64d935d692f2610e1cda/MonoGame.Framework/Net/PacketReader.cs#L84

https://github.com/MonoGame/MonoGame/blob/7c3d6870a38f8a5e479e64d935d692f2610e1cda/MonoGame.Framework/Net/PacketWriter.cs#L74

All I'm thinking is that all the existing `IPackedVector<TPacked>` implementations could continue as is but with the interface separate from `IPixel<TColor>`. I know it seems really odd but I really want to ensure that we have the greatest range of coverage possible when considering compatibility.